service/application/pom.xml

```diff
@@ -356,8 +383,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${maven-compiler-plugin.version}</version>
                 <configuration>
-                    <source>${java.version}</source>
-                    <target>${java.version}</target>
+                    <source>14</source>
+                    <target>14</target>
```

Comment:
what was the reason for this change?

Reply:
I have the same question, please address this before merging